props/blaster_BC_buttons.h

```diff
@@ -539,14 +541,42 @@ class BlasterBCButtons : public Blaster {
   }
 
 void ResetCurrentAlternative() override {
-  current_alternative = BLASTER_DEFAULT_MODE;
+  switch (blaster_mode) {
```

Comment:
why not just do:

current_alternative = blaster_mode;

?

Also, this will keep the mode when you change presets, is that the intention?

Reply:
Yes, the intent is to have not only mode persist (which it already did) but to also keep the alt sounds in sync.
I keep thinking in terms of the mode names, not as their enum equivalent to an int.
Changed, and working correctly.